framework/jaif-inference-scripts/insert-annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
Say what the script does.  The following comments says something about its implementation -- it calls another program named insert-annotations-to-source -- but not about its purpose.